ci/test.sh

```diff
@@ -72,17 +115,28 @@ endpoint=$(convox api get /apps/httpd/services | jq -r '.[] | select(.name == "w
 fetch https://$endpoint | grep "It works"
 sleep 30
 convox rack ps -a | grep httpd
+convox logs -a httpd --no-follow --since 10m
 convox logs -a httpd --no-follow --since 10m | grep service/web
 releaser=$(convox releases rollback $release -a httpd --id)
 convox ps -a httpd | grep $releaser
-ps=$(convox api get /apps/httpd/processes | jq -r '.[]|select(.status=="running")|.id' | head -n 1)
+ps=$(convox api get /apps/httpd/processes | jq -r '.[]|select(.status=="running" and .name == "web")|.id' | head -n 1)
 convox ps info $ps -a httpd | grep $releaser
 convox scale web --count 2 --cpu 192 --memory 256 -a httpd
 convox services -a httpd | grep web | grep 443:80 | grep $endpoint
 endpoint=$(convox api get /apps/httpd/services | jq -r '.[] | select(.name == "web") | .domain')
 fetch https://$endpoint | grep "It works"
-convox ps -a httpd | grep web | wc -l | grep 2
-ps=$(convox api get /apps/httpd/processes | jq -r '.[]|select(.status=="running")|.id' | head -n 1)
+# waiting for web to scale up
+i=0
+while [ "$(convox ps -a httpd | grep web | wc -l)" != "2" ]
+do
+    if [ $((i++)) -gt 50 ]; then
+        exit 1
+    fi
+    echo "waiting for web to scale up..."
+    convox ps -a httpd
+    sleep 15
+done
```

Comment:
The V3 CLI wait until the scale is complete before returning, doesn't it?
If necessary you can decrease the counter number, the current `50` It can take up to 13 minutes to return a fail.

Reply:
This one is necessary probably in gcp provider sometimes it takes a little more time.
So i am going to reduce it to 10.